### PR TITLE
Add cKES token description and icon

### PR DESCRIPTION
**Pull request by olenovyk**

### CODE REVIEW

File: src/data/mainnet/celo-tokens-info.json [hunk 1]
```diff
@@ -702,5 +702,14 @@
     "isCashInEligible": true,
     "minimumAppVersionToSwap": "1.68.0",
     "infoUrl": "https://www.coingecko.com/en/coins/tether"
+  },
+  {
+    "address": "0x456a3D042C0DbD3db53D5489e98dFb038553B0d0",
```
Comment (MuckT) [inline src/data/mainnet/celo-tokens-info.json:707]: Address should be lowercased.

Comment (MuckT) [inline src/data/mainnet/celo-tokens-info.json:707]:
```suggestion
    "address": "0x456a3d042c0dbd3db53d5489e98dfb038553b0d0",
```

**Review by MuckT — APPROVED** [review]

**Comment by MuckT** [comment]
@olenovyk could you use this asset in your PR with the correct sizing.

cKES

**Comment by olenovyk** [comment]
Hey @MuckT , thanks for the review and feedback. Pushed icon in 256x256 size. Can you please check?

**Review by MuckT — APPROVED** [review]
LGTM! 
- [cKES Contract](https://explorer.celo.org/mainnet/address/0x456a3D042C0DbD3db53D5489e98dFb038553B0d0)
- [cKES Proposal (Passed)](https://celo.stake.id/#/proposal/177)

**Pull request merged**